src/renderer/components/publication/Cover.tsx

```diff
@@ -44,16 +37,21 @@ export default class Cover extends React.Component<ICoverProps, null> {
                 }
                 authors += this.translator.translateContentField(newAuthor);
             }
+            let colors = this.props.publication.customCover;
+            if (colors === undefined) {
```

Comment:
I will merge this PR as-is, because at the moment the following bug does not exist: but, there is a potential fatal bug here (crash) when `this.props.publication.customCover` is `null` (because `colors === undefined` will not match), resulting in `colors` remaining `null`, and the rest of the code crashing. This could be detected if `strictNullChecks` was enabled in the TypeScript compiler settings (see my WIP PR), but unfortunately this would currently break a huge amount of Thorium code, so we have to accept some degree of uncertainty (no static checks), and/or rely on unit-tests, and programmer's awareness (thus why I am bringing this up here).

Reply:
I created a separate issue to track this project-wide problem. https://github.com/readium/readium-desktop/issues/634